.env.example

```diff
@@ -0,0 +1,60 @@
+# Default values for Azure OpenAI 
+# These values are configuring which models are being used by the client.
+# These default values mean that it is using the Ada 002 model for text embedding
+# and the GPT-3.5-turbo model for the prompt.
+AZURE_OPENAI_EMBEDDING_DEPLOYMENT_NAME=text-embedding-ada-002
+AZURE_OPENAI_EMBEDDING_MODEL_NAME=text-embedding-ada-002
+AZURE_OPENAI_PROMPT_DEPLOYMENT_NAME=gpt-35-turbo
+AZURE_OPENAI_PROMPT_MODEL_NAME=gpt-35-turbo
+AZURE_OPENAI_DEFAULT_ENDPOINT=https://griptape-cloud-nonprod-scus.openai.azure.com/
+
+# Get from Azure Portal https://azure.microsoft.com/en-us/get-started/azure-portal
+# These values allow the client to authenticate with Azure and use the cloud service. 
+# Input these values in order to run the Chat_Azure class. 
```

Comment:
why would I want to chat with Azure? Aren't I chatting with a bot? Is my bot using OpenAI, however?